src/utils/googleAuth.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+const GoogleAuth = (props) => {
```

Comment:
In here you'll want to make use of the `URLSearchParams` class to parse out data from the props. Check the group DM I sent out for the code example.